src/main/java/com/aws/greengrass/secretmanager/SecretManager.java

```diff
@@ -126,6 +127,15 @@ public void syncFromCloud() {
         try (LockScope ls = LockScope.lock(syncFromCloudLock)) {
             List<SecretConfiguration> secretConfiguration = getSecretConfiguration();
             localStoreMap.syncWithConfig(secretConfiguration);
+            try {
+                reloadCache();
+            } catch (SecretManagerException e) {
```

Comment:
Instead of assuming that e is caused by the local store being corrupted (and recovered from), can we put in a check to verify that?

The risk is that in the future SecretManagerException can be thrown for something that should trigger a broken state, or otherwise need different handling than just logging.